drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -32,7 +32,37 @@ matterManufacturer:
     vendorId: 0x1339
     productId: 0xAC
     deviceProfileName: plug-binary
-
+#Lifx
+  - id: 5155/169
```

Comment:
the `id` field needs to be in quotes as we've done throughout this file or else it could potentially be interpreted as an equation since this is purely numerical